profx-config.json

```diff
@@ -0,0 +1,13 @@
+{
+    "filePath" : "./testProject/src/",
+    "fileToMutate" : "HelloWorld.ts",

```

Comment:
So far only takes one file but we don't need to over complicate things until the next version I'm thinking